dataline-integrations/singer/bigquery/destination/.gitignore

```diff
@@ -0,0 +1 @@
+config/*
```

Comment:
can you configure gradle to not create a directory in the proejct path but in temp instead?

Reply:
This is for the credentials (which are not generated by gradle/tests). I'll modify this in the future PR for running these on master and building docker images.